ssc/cmod_grid.h

```diff
@@ -103,6 +93,7 @@ struct gridVariables
 			(size_t)analysis_period,
 			n_rec_lifetime,
 			curtailment_year_one,
+            scaleFactors,
```

Comment:
This line has broken running PVWatts Residential and Commercial configurations in SAM since no load_scale passed to single_year_to_lifetime_interpolated but a load passed and causes access violations in lines 86 and 93 in lib_time.cpp